### DISCUSSION TITLE
How to get webcam information

### DISCUSSION THREAD

**Discussion by Luxjacek** [roleoroleo/yi-hack-MStar#527] [General]
Hey, I lost my mount holder from my webcam Yi, so i do not know, what is exactly the nubmer of my webcam, so that i can not hack it. Is there a soluton to get all information about webcam ?

**Comment by roleoroleo** [comment]
It depends on the platform.
If it's a MStar, the only way to enter the cam is opening it and using a rs232 adapter.
If it's an Allwinner (or Allwinner-v2) cam, the backup procedure is able to extract all you need.

In general, if you open it and take some picture, we can try to guess.

**Reply by Luxjacek** [reply]
Thank You so much. I found out, that 2 years ago i was havig this software and it was working LFUSY6073L
Maybe thanks that, You will be able to give me a hint wich repository i should download ?

**Reply by roleoroleo** [reply]
Sorry but the serial number is not enough to guess the model.
You would need the fw version.

**Comment by Luxjacek** [comment]
> (or Allwinner-v2) cam, the backup procedure is able to extract all you need.
Lets try. I did backup. And i have there some folders. homever - 9.0.27.20_202108161046,  and midblock0 - till 7. What do i have to give you to guess the version ?

**Reply by roleoroleo** [reply]
LFUS with 9.0.27.20_202108161046 means h60ga (allwinner-v2).

**Comment by Luxjacek** [comment]
Dear roleoroleo, i bought 2 cameras, but need to know if its hackable ? It works on Yi IoT app, here some pictures.
https://photos.app.goo.gl/eLoGf27CjbFfWrm1A
and this is MAC results 
![image](https://github.com/user-attachments/assets/e25c8b71-d20f-4492-b8a2-328eb7ff529d)

**Reply by roleoroleo** [reply]
Please, take a picture of the cpu.
Are you able to read the fw version inside the app?
